projects/ancestor/ancestor.py

```diff
@@ -1,3 +1,111 @@
+class Stack():
+    def __init__(self):
+        self.stack = []
+
+    def push(self, value):
+        self.stack.append(value)
+
+    def pop(self):
+        if self.size() > 0:
+            return self.stack.pop()
+
+        raise IndexError("cannot pop empty stack")
+
+    def size(self):
+        return len(self.stack)
+
+
+class Graph():
+    def __init__(self):
+        self.vertices = {}
+
+    def add_vertex(self, value):
+        self.vertices[value] = set()
+
+    def isvertex(self, value):
```

Comment:
```suggestion
    def is_vertex(self, value):
```
just a suggestion. Looks good though!